packages/jitm/src/Manager.php

```diff
@@ -177,7 +181,7 @@ function jitm_enqueue_files() {
 
 		wp_enqueue_script(
 			'jetpack-jitm-new',
-			Jetpack::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
+			\Jetpack::get_file_url_for_environment( '_inc/build/jetpack-jitm.min.js', '_inc/jetpack-jitm.js' ),
```

Comment:
Would it be a good idea to have a new assets class to manage different types of assets?

Reply:
`Jetpack` would not be present if the package is not running alongside with Jetpack